### PR TITLE
roadmap updates

### PR DESCRIPTION
**Pull request by prasek**
### Description of your changes
 - Update roadmap.md with San Diego milestone epics and futures

### Checklist
I have:
- [ ] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Ensured this PR contains a neat, self documenting set of commits.
- [ ] Updated any relevant [documentation], [examples], or [release notes].
- [ ] Updated the RBAC permissions in [`clusterrole.yaml`] to include any new types.

[skip ci]

### CODE REVIEW

**Comment by prasek** [comment]
@jbw976 added as WIP, scoping first sounds good

**Comment by prasek** [comment]
@jbw976 updated to reflect what's in v0.4.0, think it's ready to go.

File: ROADMAP.md [hunk 1]
```diff
@@ -82,52 +82,74 @@ Any dates listed below and the specific issues that will ship in a given milesto
   * AWS Infra Stack: single-region secure connectivity: EKS & RDS, ElastiCache, Buckets [#616](https://github.com/crossplaneio/crossplane/issues/616)
   * Azure Infra Stack: single-region secure connectivity: AKS & AzureSQL, AzureCache, Buckets [#617](https://github.com/crossplaneio/crossplane/issues/617)
   * Stacks v1 CLI / kubectl plugin: init, build, push commands [#614](https://github.com/crossplaneio/crossplane/issues/614)
-
+u
```
Comment (jbw976) [inline ROADMAP.md]: how'd this little straggler make it in here? please remove 😆

File: ROADMAP.md [hunk 2]
```diff
@@ -82,52 +82,74 @@ Any dates listed below and the specific issues that will ship in a given milesto
...
+  * Sample infra and app repos you can `kubectl apply -f` and have a working environment quickly
+    * infrastructure (networks, subnets, managed k8s cluster, resource classes for databases, etc.)
+    * apps (e.g. kubernetes core resources for e.g. a Wordpress app plus the resource claims for managed service dependencies
+  * Update crossplane.io services guides and stacks guides to use `kubectl apply -f` technique
+
+* Claim-based provisioning of [Rook](https://rook.io/)-managed databases [#862](https://github.com/crossplaneio/crossplane/issues/862)
+  * Support for CockroachDB and Yugabyte DB
+
+ * Release automation for shorter release cycles and hot fixes [#864](https://github.com/crossplaneio/crossplane/issues/864) 
+   * Updating pipelines to include automation #6
```
Comment (jbw976) [inline ROADMAP.md]: all of these issue links underneath release automation are not showing as linkable, they aren't linking to the full issue path like other items in this roadmap are.  e.g. update `#6` to:
```
 [#6](https://github.com/crossplaneio/crossplane/issues/6)
```

File: ROADMAP.md [hunk 3]
```diff
@@ -82,52 +82,74 @@ Any dates listed below and the specific issues that will ship in a given milesto
...
+   * SonarCloud checks for cloud provider stacks #875
+   * crossplane-runtime build pipelines crossplaneio/crossplane-runtime#14
+
+ * Trace utility for enhanced debugging support. [#744]
```
Comment (jbw976) [inline ROADMAP.md]: also a broken link

File: ROADMAP.md [hunk 4]
```diff
@@ -82,52 +82,76 @@ Any dates listed below and the specific issues that will ship in a given milesto
...
+ * Release automation for shorter release cycles and hot fixes [#864](https://github.com/crossplaneio/crossplane/issues/864) 
+   * Updating pipelines to include automation #6
+   * SonarCloud checks for cloud provider stacks #875
+   * crossplane-runtime build pipelines crossplaneio/crossplane-runtime#14
+
+ * Trace utility for enhanced debugging support. [#744] 
+
+ * Simple Resource Class Selection [#952]
+
+## Under consideration v0.5.0
```
Comment (jbw976) [inline ROADMAP.md]: want to also add "default classes of service" that provide sane and reasonable defaults to remove initial setup friction and get started even quicker?

and is that related to example config repos / best practice repos?

File: ROADMAP.md [hunk 5]
```diff
@@ -82,52 +82,76 @@ Any dates listed below and the specific issues that will ship in a given milesto
...
+   * Add support for MySQL, Redis, Buckets, and more. (GitLab 12.6)
+
+ * v1beta2 Services APIs
+   * Incorporate beta1 feedback
+   * Upgrade Azure stack to v1beta2 (AzureDB, Azure Cache for Redis)
+     * crossplaneio/stack-azure#28 Azure SQL and Redis resources v1beta1
+   * Upgrade other supported services to v1beta2 (e.g. Buckets, etc.)
+   * Code generation of API types, controller scaffolding to further streamline additional services
+
+* More real-world Stacks into multiple clouds
```
Comment (jbw976) [inline ROADMAP.md]: further enhancements to security and permissions/roles model?  (maybe include in extended roadmap, not v0.5.0)

**Comment by prasek** [comment]
@jbw976 updated, can you merge this if it looks good to you?

**Comment by jbw976** [comment]
Thanks for making these updates to the roadmap @prasek! 🚀

**Pull request merged**